docs/volto_customization/installation.md

```diff
@@ -17,150 +17,123 @@ Getting started with Volto involves setting up a development environment, unders
 
 ## Prerequisites
 
-Before you start working with Volto, ensure you have the following prerequisites:
+Before you start working with this training, ensure you have the following prerequisites:
 
-- <a target="_blank" href="https://nodejs.org/en">Node.js LTS (16.x)</a> - (<a target="_blank" href="https://6.docs.plone.org/install/install-from-packages.html#nvm">see instructions for installation</a>)
+- <a target="_blank" href="https://nodejs.org/en">Node.js LTS (>=20.x)</a> - (<a target="_blank" href="https://6.docs.plone.org/install/install-from-packages.html#nvm">see instructions for installation</a>)
 - <a target="_blank" href="https://www.python.org/">Python</a> - See below for specific versions.
+- <a target="_blank" href="https://pipx.pypa.io/stable/">pipx</a>
+- <a target="_blank" href="https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating">nvm</a>
+- <a target="_blank" href="https://www.gnu.org/software/make/">GNU make</a>
 - <a target="_blank" href="https://www.docker.com/get-started">Docker</a> (if using the Plone docker images - <a target="_blank" href="https://6.docs.plone.org/install/containers/index.html">see instructions for installation and usage</a>)
 
 The versions of Python that are supported in Volto depend on the version of Plone that you use.
 
 | Plone | Python       | Volto        |
 | ----- | ------------ | ------------ |
-| 6.0   | 3.8-3.11     | 16.0 or 17.0 |
+| 6.0   | 3.8-3.12     | 17.0 or 18.0 |
```

Comment:
@nileshgulia1 this is not accurate, 6.0 is "LTS" so the recommended version is Volto 16.0.0 which is also "LTS".

Reply:
fixed in [d91612d](https://github.com/plone/training/pull/860/commits/d91612d721ed191d3f0935f6a5758754357f06f7)